docs/content/items/routeros.md

```diff
@@ -16,13 +16,23 @@ Manages RouterOS configuration through the API (port 8728). You can address ever
         "/interface/vlan?name=vlan7": {
             "delete": True,
         },
+        "/interface/vlan": {
+            "purge": {
+                "id-by": "name",
```

Comment:
The `purge` mechanism could use some more documentation. This is something that we came up with, so we can’t refer people to Mikrotik docs.

The docs should answer the following questions:

- How is `id-by` used? Which value must I use, how do I find it?
- What’s the meaning of `keep/dynamic`? Are there more options for `keep` other than `dynamic`?

Reply:
Indeed, I will extend the docs on this.

Reply:
Would you mind re-reading the documentation? Thanks in advance!